.github/workflows/code-checks.yml

```diff
@@ -96,3 +96,30 @@ jobs:
         with:
           token: ${{secrets.CODECOV_TOKEN}}
           files: ./app/mobile/coverage/.tmp/coverage-0.json
+
+  lingui-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - uses: ./.github/actions/provision
+
+      - name: Run lingui
+        working-directory: ./apps/mobile
+        run: pnpm lingui compile --strict
+      # TODO ENG-14 - this should only run on changes to the mobile app
```

Comment:
I tried to make it run only on changes to `apps/mobile` but couldn't get it working. 

I can remove this / try improve it by fixing it